tests/routes/test_raster_tiles.py

```diff
@@ -41,29 +45,122 @@ def test_dynamic_tiles(x, y, multiplier, client):
         saved_bytes.seek(0)
         _check_png(saved_bytes, multiplier)
     finally:
-        log_client = boto3.client(
-            "logs", region_name="us-east-1", endpoint_url=AWS_ENDPOINT_URI
+        _get_logs()
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize("x, y, confirmed_only", [(0, 0, False), (0, 0, True)])
+def test_dynamic_tiles_params(x, y, confirmed_only, client):
+    """
+    Test dynamic tile cache end to end
+    """
+    try:
+        response = client.get(
+            f"/wur_radd_alerts/v20201214/dynamic/14/{x}/{y}.png",
+            params={"confirmed_only": confirmed_only},
+            stream=True,
         )
-        log_group_name = "/aws/lambda/test_project-lambda-tiler"
-        for log_stream in log_client.describe_log_streams(logGroupName=log_group_name)[
-            "logStreams"
-        ]:
-            log_events = log_client.get_log_events(
-                logGroupName=log_group_name, logStreamName=log_stream["logStreamName"],
-            )["events"]
 
-            for event in log_events:
-                # for some reason stack traces come with carriage returns,
-                # which overwrites the line instead of making a new line
-                message = event["message"].replace("\r", "\n")
-                print(f"{log_stream['logStreamName']}: {message}")
+        assert (
+            response.status_code == 200
+        ), f"Bad response for request {response.request.url}: {response.json()}"
+
+        img = _response_to_img(response)
+        _check_filtered_png(img, confirmed_only)
+
+        # check if s3 file copied. It should now be accessible using the default endpoint.
+        # saved_bytes = BytesIO()
+        # s3_client = boto3.client("s3", endpoint_url=AWS_ENDPOINT_URI)
+        # s3_client.download_fileobj(
+        #     "gfw-tiles-test",
+        #     f"wur_radd_alerts/v20201214/default/14/{x}/{y}.png",
+        #     saved_bytes,
+        # )
+        # saved_bytes.seek(0)
+        # _check_png(saved_bytes, multiplier)
```

Comment:
Was this check waiting on some code which is now finished, or is it still meant to be commented?